catroid/src/org/catrobat/catroid/content/bricks/BrickBaseType.java

```diff
@@ -47,6 +51,89 @@
 	protected transient int alphaValue = 255;
 	public transient boolean animationState = false;
 
+	protected boolean commentedOut;
+
+	@Override
+	public boolean isCommentedOut() {
+		return commentedOut;
+	}
+
+	@Override
+	public void setCommentedOut(boolean commentedOut) {
+		this.commentedOut = commentedOut;
+		setCommentedOutAppearance();
+	}
+
+	public void setCommentedOutAppearance() {
+		if (view == null) {
+			return;
+		}
+
+		if (commentedOut) {
+			grayscaleBackground();
+		} else {
+			undoGrayscaleBackground();
+		}
+		doPadding();
+	}
+
+	private void grayscaleBackground() {
+
+		ViewGroup viewGroup = (ViewGroup) view;
+		for (int index = 0; index < viewGroup.getChildCount(); index++) {
+			View child = viewGroup.getChildAt(index);
+
+			if (child instanceof BrickLayout) {
+				Drawable background = child.getBackground();
+
+				if (background == null) {
+					return;
+				}
+
+				ColorMatrix matrix = new ColorMatrix();
+				matrix.setSaturation(0);
+
+				ColorMatrixColorFilter filter = new ColorMatrixColorFilter(matrix);
+
+				background.setColorFilter(filter);
+			}
+		}
+	}
+
+	private void undoGrayscaleBackground() {
+
+		ViewGroup viewGroup = (ViewGroup) view;
+
+		for (int index = 0; index < viewGroup.getChildCount(); index++) {
+			View child = viewGroup.getChildAt(index);
+
+			if (child instanceof BrickLayout) {
+				Drawable background = child.getBackground();
+				if (background != null) {
+					background.clearColorFilter();
+				}
+			}
+		}
+	}
+
+	public void doPadding() {
+		if (adapter == null) {
+			return;
+		}
+
+		int next = adapter.getBrickList().indexOf(this) + 1;
+		boolean hasNext = next < adapter.getBrickList().size();
+		boolean nextCommentStateDifferent = hasNext && (this.isCommentedOut() != adapter.getBrickList().get(next).isCommentedOut());
+		boolean nextIsScriptbrick = hasNext && adapter.getBrickList().get(next) instanceof ScriptBrick;
+
+		int paddingLeft = isCommentedOut() ? 75 : 0;
+		if (nextCommentStateDifferent && !nextIsScriptbrick) {
```

Comment:
maybe do the same as above like
`int paddingBottom = nextCommentStateDifferent && !nextIsScriptbrick ? 50 : 0;
view.setPadding(paddingLeft, view.getPaddingTop(), 0, paddingBottom);`

would save some lines ;)